touchforms/backend/xformplayer.py

```diff
@@ -718,9 +720,17 @@ def current_question(xform_session, override_state=None):
 
 
 def heartbeat(session_id):
-    # just touch the session
-    with global_state.get_session(session_id) as xfsess:
-        return {}
+    """
+    The heartbeat will update the session's last seen timestamp and sequence ID.
+    """
+    # todo: cz as of 8/5/2015 I'm pretty sure this isn't necessary. persistence can
+    # just pop this back into the context whenever necessary so no longer super relevant
+    # to keep around in memory while the tab is open.
+    with global_state.get_lock(session_id):
+        # this updates the session sequence ID and timestamp and persists it
+        with global_state.get_session(session_id):
```

Comment:
we can totally get rid of this function as far as i know. we only make a request to `heartbeat` via the `restarter.sh` and that call doesn't include a session id so it returns a 400 response and this code never executes. ideally, this function should just return a message stating the server is up and a response 200